src/main/java/org/opensearch/plugin/insights/core/service/categorizer/SearchQueryCounters.java

```diff
@@ -23,8 +23,12 @@
  */
 public final class SearchQueryCounters {
     private static final String LEVEL_TAG = "level";
-    private static final String TYPE_TAG = "type";
+    private static final String QUERY_TYPE_TAG = "type";
     private static final String UNIT = "1";
+    private static final String UNIT_MILLIS = "ms";
+    private static final String UNIT_CPU_CYCLES = "ns";
+    private static final String UNIT_BYTES = "bytes";
```

Comment:
Right now this is all in memory. None of these metrics are serialized over the wire, right? Else, we need to think about the backward compatibility aspects

Reply:
Yup